queries.sql

```diff
@@ -231,4 +231,49 @@ GROUP BY
 ORDER BY
     frequency DESC
 LIMIT
-    1;
\ No newline at end of file
+    1;
+
+-- * Project 4
+SELECT
+    animals.name,
+    vets.name,
+    visited
+FROM
+    animals,
+    visits,
+    vets
+WHERE
+    animals.id = visits.animal_id
+    AND vets.id = visits.vet_id
+    AND vets.name = 'William Tatcher'
+ORDER BY
+    visited DESC
+LIMIT
+    1;
+
+SELECT
+    animals.name animals,
+    vets.name vets,
+    COUNT(animals)
+FROM
+    animals,
+    visits,
+    vets
+WHERE
+    animals.id = visits.animal_id
+    AND vets.id = visits.vet_id
+    AND vets.name = 'Stephanie Mendez'
+GROUP BY
+    animals.name,
+    vets.name,
+    visits.visited
+ORDER BY
+    visited DESC;
+
+SELECT
+    species.name spec_names,
+    vets.name
+FROM
+    species,
+    specializations FULL
+    JOIN vets ON vets.id = specializations.vet_id;
```

Comment:
Your screenshots 👀
![image](https://user-images.githubusercontent.com/78251927/236267897-f87b207a-70fd-4d6e-854c-ec43dc9ebd52.png)

According to the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md) please kindly write queries to answer all the following and add the screenshot with the results of your queries to your Pull Request description so that we can verify if the results are correct 🤗🛠

![image](https://user-images.githubusercontent.com/78251927/236267641-5bd731f8-86e6-4c47-a9cc-87a07fecee4b.png)